public/app/features/dashboard-scene/edit-pane/DashboardEditPane.tsx

```diff
@@ -53,25 +54,34 @@ export class DashboardEditPane extends SceneObjectBase<DashboardEditPaneState> {
   }
 
   public selectObject(obj: SceneObject, id: string, multi?: boolean) {
-    const currentSelection = this.state.selectedObject?.resolve();
-    if (currentSelection === obj) {
+    const prevItem = this.state.selection?.getFirstObject();
+    if (prevItem === obj && !multi) {
       this.clearSelection();
```

Comment:
feels like this is not working like it should, I was unable to unselect an item by selecting it again. This is how it usually works in multi select systems, if I hold down shift and click a selected object again (independent of the order clicked) I should be able to unselect it. 

Test this PR and was unable to unselect a selected item by selecting it again

Reply:
ah yes, that doesn't work indeed. I'll push a fix with this, thank you!

Reply:
yes, it's working now!